### DISCUSSION TITLE
Is it possible to use MSBuild v17.4 for net461 and netcore3.1?

### DISCUSSION THREAD

**Discussion by rstm-sf** [dotnet/msbuild#8235] [Q&A] [answered]
It is mentioned [here](https://github.com/dotnet/msbuild/tree/main/src/Utilities#netstandard20-target) that for net6.0 we can use net7.0 libraries. Is it possible to use for lower targets?

An example where tests break because they don't see Microsoft.Build.Utilities.Core https://github.com/ashmind/mirrorsharp/pull/190/files#diff-5f827a8e81001a61ff42f6966d224b206eb04ea3c2e18aec2aeddac04e401510R18

**Comment by JanKrivanek** [comment, marked as answer]
Unfortunately as mentioned in the referenced doc - the netstandard2.0 version has a specific usage only, otherwise the respective .NET and .NET Framework versions of packages need to be used (and they cannot be used from older runtime).
